services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslMovementSpec.groovy

```diff
@@ -104,6 +109,15 @@ class IslMovementSpec extends BaseSpecification {
         !islUtils.getIslInfo(allLinks, expectedIsl).present
         !islUtils.getIslInfo(allLinks, islUtils.reverseIsl(expectedIsl)).present
 
+        and: "Self-loop rule packet counter is incremented and logged in otsdb"
+        def statsData = null
+        Wrappers.wait(70, 2) {
```

Comment:
What about moving 70 to constants? In the future, it will be easier to tune this time.

STATS_LOGGING_TIMEOUT = 70.

Reply:
done